clang/include/clang/Sema/Sema.h

```diff
@@ -9713,7 +9713,8 @@ class Sema final {
   /// not already done so.
   void DeclareImplicitDeductionGuides(TemplateDecl *Template,
                                       SourceLocation Loc);
-  FunctionTemplateDecl *DeclareImplicitDeductionGuideFromInitList(
+  /// Declare aggregate deduction guides.
```

Comment:
I'm not sure this documentation is going to help anyone after looking at the name of function.

Reply:
OK, I removed it.